sdcm/coredump.py

```diff
@@ -390,7 +390,21 @@ def get_list_of_cores_json(self) -> Optional[List[CoreDumpInfo]]:
                 pid_list.append(CoreDumpInfo(pid=str(dump['pid']), node=self.node))
         return pid_list
 
+    def _wait_for_pending_coredumps(self):
+        """
+        Check for active coredumps being created by waiting for systemd-coredump@ to stop running.
+        """
+        result = self.node.remoter.run('systemctl list-units --type=service --state=running | grep -q "systemd-coredump@"',
+                                       verbose=False, ignore_status=True)
+        if result.ok:
+            self.log.debug("Pending coredumps found, waiting for them to complete")
```

Comment:
I think it worth being an `info`  log message so we could read it in jenkins logs first.

Reply:
changed